### ISSUE TITLE
jQuery problem

### ISSUE THREAD

**Issue by remontees** [tiff/wysihtml5#56]
Here is the error (copy from firebug) :
    c is undefined
hasElementWithClassName(c=undefined)wysiht....min.js (ligne 134)
_initSandbox()wysiht....min.js (ligne 219)
constructor(a=Object { _constructing=true, config={...}, events={...}, more...}, b=undefined, c=Object { style=true, toolbar="undefined-wysihtml5-toolbar", autoLink=true, more...})wysiht....min.js (ligne 216)
extend()wysiht....min.js (ligne 105)
extend()wysiht....min.js (ligne 103)
constructor(a=undefined, d=Object { toolbar="undefined-wysihtml5-toolbar", parserRules={...}})wysiht....min.js (ligne 257)
extend()wysiht....min.js (ligne 103)
e(a=undefined)bootst....min.js (ligne 1)
e(b=[label.wysihtml], c=undefined)bootst....min.js (ligne 1)
e()bootst....min.js (ligne 1)
f(a=[label.wysihtml, textarea#rms_newsbundle_articletype_contenu.wysihtml], c=function(), d=undefined)jquery.min.js (ligne 2)
f(a=function(), b=undefined)jquery.min.js (ligne 2)
e(b=undefined)bootst....min.js (ligne 1)
()ajouter (ligne 87)
[Stopper sur une erreur]    

...fier=a++))+":"+e,h=c[f];h||(h=c[f]=d.getElementsByClassName(e));return 0<h.lengt...

wysiht....min.js (ligne 134)

**Comment by tiff** [comment]
This doesn't help me much. Can you please give some additional informations?

**Comment by remontees** [comment]
The source code of the page :
<!DOCTYPE html>
<html lang="fr">
    <head>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <title> Ajout d'un article -    News  - Remontées mécaniques

</title>
                    <link rel="stylesheet" title="Bootstrap2" href="/Symfony/web/css/bootstrap.css" type="text/css" />
                <link rel="shortcut icon" type="image/x-icon" href="/Symfony/web/favicon.ico" />
    </head>
    
    
    
    
    <body>
        <header role="banner" class="navbar navbar-inner">
            <nav role="navigation" class="container">
                <a href="/" title="Retour accueil" id="logo" class="brand"><img src="/Symfony/web/img/logo.png" alt="Logo du site" /></a>
                <div id="member">
                                            <span>remontees</span><br />
                        <a href="/Symfony/web/logout" title="Déconnexion">Déconnexion</a>
                                    </div>
                <div class="nav-collapse">
                <ul class="nav">
                    <li><a href="/Symfony/web/news">News</a></li>
                    <li><a href="#">Reportages</a></li>
                    <li><a href="#">Dossiers explicatifs</a></li>
                    <li><a href="#/">Forum</a></li>
                </ul>
                </div>
            </nav>
        </header>
        <section id="content" role="main" class="container">
                <h1>News</h1>

```
    <h2>Ajouter un article</h2>
```

<div class="well">
    <form method="post" >
        <div>
        <input type="hidden" id="rms_newsbundle_articletype__token" name="rms_newsbundle_articletype[_token]" value="fa960a96125af155085977cd3cb39a4db2d77e72" /><div><label class=" required">Date</label><div id="rms_newsbundle_articletype_date"><div id="rms_newsbundle_articletype_date_date"><input type="text" id="rms_newsbundle_articletype_date_date_day" name="rms_newsbundle_articletype[date][date][day]" required="required" value="8" /><input type="text" id="rms_newsbundle_articletype_date_date_month" name="rms_newsbundle_articletype[date][date][month]" required="required" value="5" /><input type="text" id="rms_newsbundle_articletype_date_date_year" name="rms_newsbundle_articletype[date][date][year]" required="required" value="2012" /></div><div id="rms_newsbundle_articletype_date_time"><input type="text" id="rms_newsbundle_articletype_date_time_hour" name="rms_newsbundle_articletype[date][time][hour]" required="required"    size="1" value="18" />:<input type="text" id="rms_newsbundle_articletype_date_time_minute" name="rms_newsbundle_articletype[date][time][minute]" required="required"    size="1" value="43" /></div></div></div><div><label for="rms_newsbundle_articletype_titre" class=" required">Titre</label><input type="text" id="rms_newsbundle_articletype_titre" name="rms_newsbundle_articletype[titre]" required="required" /></div><div><label class="wysihtml required" placeholder="Commencez à écrire votre texte..." for="rms_newsbundle_articletype_contenu">Contenu</label><textarea id="rms_newsbundle_articletype_contenu" name="rms_newsbundle_articletype[contenu]" required="required"    class="wysihtml" placeholder="Commencez à écrire votre texte..."></textarea></div><div><label class=" required">Tags</label><div id="rms_newsbundle_articletype_tags"></div></div>
        <a href="#" id="add_tag">Ajouter un tag</a>
        </div>
        <input type="submit" class="btn btn-primary" />
    </form>
</div>

```
<p>
    Rédigez bien la news !
</p>
<p>
    <a href="/Symfony/web/news" class="btn">
        <i class="icon-chevron-left"></i>
        Retour à l'accueil des news
    </a>
</p>
    </section>
    <footer role="contentinfo">
    <section class="container">
        <div class="haut-footer">
            <div class="span4">
                © 2011 - 2012<br />
                michel
            </div>
            <div class="span4">
                <h6>S'inscrire à la newsletter</h6>
                Formulaire à venir...
            </div>
            <div class="span4">
                <h6>Suivez-nous !</h6>
                <img src="/Symfony/web/img/flux.png" title="Flux RSS" alt="Flux RSS" />
            </div>
        </div>
    </section>
    </footer>
    <script type="text/javascript" src="/Symfony/web/js/wysihtml5-0.3.0_rc3.min.js"></script>
        <!--[if lt IE 9]>
            <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
        <![endif]-->
        <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js"></script>
        <script type="text/javascript" src="/Symfony/web/js/bootstrap.js"></script>
```

<script type="text/javascript">
    $(".alert").alert();
</script>


<script type="text/javascript" src="/Symfony/web/js/prettify.js"></script>

<script type="text/javascript" src="/Symfony/web/js/bootstrap-wysihtml5.min.js"></script>

<script type="text/javascript">
$('.wysihtml').wysihtml5();
</script>

<script type="text/javascript" charset="utf-8">
    $(prettyPrint);
</script>

```
</body>
```

</html>

**Comment by tiff** [comment]
I think can't help you with this one. It doesn't look like a wysihtml5 problem... I haven't heard from anyone else who has a similar problem.
The stack trace you provided shows that a function named `hasElementWithClassName` is called within wysihtml5.views.Composer.prototype._initSandbox but if you check the source code of wysihtml5 there's no function called with such a name.
